.github/workflows/codeql.yml

```diff
@@ -0,0 +1,93 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches:
+    - main
+    - release-1[1-9].[0-9]
+    - codeql-workflow # TODO: remove this after PR is complete
+  schedule:
+    - cron: '59 23 * * *'
+  workflow_dispatch:
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-22.04
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'cpp', 'python']
```

Comment:
We could easily drop python from this list, as it does not look like it adds some value.

Reply:
Python now completes in insignificantly small time. No need to drop that I guess.